lib/perl/Genome/SoftwareResult/User.pm

```diff
@@ -39,5 +43,86 @@ class Genome::SoftwareResult::User {
     doc => 'links a software result to other entities which depend on it',
 };
 
+sub with_registered_users {
+    my $class = shift;
+    my %params = Params::Validate::validate(
```

Comment:
I agree, I was under the impression from the documentation that it was only a module-specific setting. If that's not the case, I'll go ahead and make the change!